### PR TITLE
[SPARKTA-442] [SPARKTA-448] [SPARKTA-450] [SPARKTA-452] [DRIVER] [PLUGINS] [DIST] Cluster execution running over Mesos and Yarn with fatJar and Plugins in one module 

### PR DESCRIPTION
**Pull request by compae**
#### Description

Removed HDFS Utils and uploaded files in cluster execution. Classpath and plugins are not necessary.

One fatJar with the driver dependencies are sufficient.

All plugins modules are flattened in one module and serving-api and driver have this dependency.

All addToClasspath are been removed from cluster execution.

Now we support user plugins with one list of jars path files in the policy.

Bugs corrected.

### CODE REVIEW

File: driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala [hunk 1]
```diff
@@ -89,7 +68,7 @@ object SparktaClusterJob extends SparktaSerializer {
 
         val streamingContextService = new StreamingContextService(Some(policyStatusActor))
         val ssc = streamingContextService.clusterStreamingContext(
-          policy, pluginsClasspathFiles, Map("spark.app.name" -> s"${policy.name}")).get
+          policy, Seq(), Map("spark.app.name" -> s"${policy.name}")).get
```
Comment (anistal) [inline driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala]: If always there is an empy Seq() in the sign of the method remove it from the function.

File: serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/ClusterLauncherActor.scala [hunk 2]
```diff
@@ -126,25 +100,49 @@ class ClusterLauncherActor(policy: AggregationPoliciesModel, policyStatusActor:
...
-    spark.waitFor()
-    val exit = spark.exitValue()
-    log.debug(s"Spark exit status: $exit")
-    if (exit != 0) setErrorStatus
+
+    log.info(s"$sparkHome/bin/spark-submit --master $master " ++ args.foldLeft("") {
+      (s: String, pair: (String, String)) =>  s"$s ${pair._1} ${pair._2}"
+    })
+
+    Future[(Int, Process)] {
```
Comment (anistal) [inline serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/ClusterLauncherActor.scala]: Is it necessary that this block of code is asynchronous?. I prefer not to use futures here (is is only my personal opinion).

File: serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/ClusterLauncherActor.scala [hunk 3]
```diff
@@ -126,25 +100,49 @@ class ClusterLauncherActor(policy: AggregationPoliciesModel, policyStatusActor:
     } else false
 
   private def launch(main: String, hdfsDriverFile: String, master: String, args: Map[String, String],
-                     driverParams: Seq[String]): Unit = {
+                     driverParams: Seq[String], clusterFiles: Seq[String]): Unit = {
     val sparkLauncher = new SparkLauncher()
       .setSparkHome(sparkHome)
       .setAppResource(hdfsDriverFile)
       .setMainClass(main)
       .setMaster(master)
```
Comment (danielcsant) [inline serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/ClusterLauncherActor.scala]: add `.setVerbose(true)` please

File: dist/src/main/resources/application.conf [hunk 4]
```diff
@@ -23,23 +23,22 @@ sparkta {
     spark.app.name = SPARKTA
     spark.sql.parquet.binaryAsString = true
     spark.streaming.concurrentJobs = 1
```
Comment (danielcsant) [inline dist/src/main/resources/application.conf]: leave these commented lines please. They are useful for the user when he wants to launch the benchmark

**Comment by danielcsant** [comment]
+1

**Comment by anistal** [comment]
+1

**Pull request merged**